ActindoWorkFlow/Operation/02_TrackWorkflowProcess.md

```diff
@@ -0,0 +1,245 @@
+
+[comment]: <> (Useful for user to mention those options? Is it possible to configure it somehow?)
+
+6. Check the queue type in the *Queue type* column.  
+The *Default* queue type is normally selected. For detailed information about configuring queue types, see [Configure the queue types](#to-be-competed).
+
+  > [Info] A different queue type can be selected by editing an action, see [Edit an action](01_ManageWorkflows.md#edit-an-action).
+
+7. Check the next point in time when the action is being tried in the *Defer until* column.
+
+[comment]: <> (Useful for user to mention it? Is it possible to configure it somehow?)
+
+
+---
+[comment]: <> (Further info probably only for Troubleshooting)
+
+
+- If an error is displayed, take the necessary action to solve the problem before retrying the action.
+
+  > [Info] Further information about the error displayed can be found in the *Logs* tab.
+
+- Click the checkbox to select the action to be checked.    
+The (editing) toolbar is displayed.
+
+  > [Info] If several action have been filtered according to status, it is also possible to select them all simultaneously. Click the checkbox in the column header to select all actions filtered. For detailed information about this process, see [Retry a process action](#to-be-completed).
+
+- Click the [RETRY] button to execute the action again.
+
+
+
+### Next steps
+
+- [Check the token status and content](#check-the-token-status-and-content)
+
+### See also
+
+- [Troubleshooting](#to-be-determined)
+
+## Check the token status and content
+
+The *Tokens* tab displays the tokens that are being or have been processed. A token is a piece of data needed to complete an action, for example a document. When a transaction is performed, a token is generated, for instance an invoice, which in turn initiates the workflow process.
+
+A process is started with one token at the start point, which will be processed in each action executed and, if necessary, turned into several tokens. It is therefore possible that a place contains several tokens, for example, several delivery notes issued for different parts of a same order. Each token will then be processed separately, that means, that the subsequent actions will be executed once for each available token.
+
+### Prerequisites
+
+- A workflow is created, see [Create a workflow](01_ManageWorkflows.md#create-a-workflow).
+- A workflow process has been initiated, see [Link](#to-be-determined).
+
+
+### Procedure
+
+*Workflows > Menu entry Processes > Tab OVERVIEW > Select process > Tab Tokens*
+
+[comment]: <> (Specify Tab Tokens or start with OVERVIEW -> clicking on place shows tokens too?)
+
+![Workflow](/Assets/Screenshots/ActindoWorkFlow/Workflows/Tokens.png "[Workflow]")
+
+1. Click a place in the workflow process diagram to display the available token(s).  
+The available token(s) in the selected place are displayed in the *Tokens* tab.
```

Comment:
tokens werden nur angezeigt, wenn diese den place bereits durchlaufen haben/in progress sind